src/orchard/inspect.clj

```diff
@@ -167,16 +179,24 @@
         (s/join sep)
         (format fmt))))
 
-(defn- short? [coll]
-  (<= (count coll) 5))
+(def ^:private ^:dynamic *max-atom-length* 150)
+(def ^:private ^:dynamic *max-coll-size* 5)
```

Comment:
Why is one size and the other length?

Reply:
Hah I think these names are a sign of my java heritage, and am happy to adjust them if needed. I started with max-string-length and max-coll-count, but "count" sounded weird to me because it's more about "how much of this collection do I want to see" vs what could be taken as "how many collections do I want to see". It turned out "string" was too specific, so it became `max-atom-length`, and java collections have sizes, so I ended up with `max-coll-size`.
Later on, I realized it's really more like `max-branching-factor`, and that it would be cool, if instead of having to set it manually, I could just tell inspector how much data I want to see, and have it go figure reasonable depths and breadths on a per-node basis. That'd be for another time though!

Reply:
Naming is hard. :-) Thanks for explaining your reasoning. It makes sense to me.